### ISSUE TITLE
Typing error, setSortConfig instead of setSortConfigs

### ISSUE THREAD

**Issue by AngelFraga** [handsontable/handsontable#5545]
I saw typo errors in `Handsontable.d.ts` file: 

- `MultiColumnSorting.setSortConfigs` should be `MultiColumnSorting.setSortConfig`
-  `ColumnSorting.setSortConfigs` should be `ColumnSorting.setSortConfig` .

After this fix, interface will fit with the official documentation here `https://handsontable.com/docs/6.1.1/demo-sorting.html`.

* Handsontable version: latest

**Comment by AMBudnik** [comment]
Yes, you're right.

Thank you for sharing @AngelFraga

**Comment by AngelFraga** [comment]
Thanks for check it 👍 
Until the fix it, the easy workaround is: 

```
...
const pl = htInstance.getPlugin('columnSorting')
 as Handsontable.plugins. ColumnSorting;
pl['setSortConfig']();
...
```

**Comment by AMBudnik** [comment]
> Logging out the currentSortConfig and destSortConfig results in the first argument being the Hansontable instance and the second argument being the current sort config.

src: https://forum.handsontable.com/t/aftercolumnsort-parameters-dont-seem-to-match-documentation-or-typings-file/2591

we should also attach those changes.

**Comment by AMBudnik** [comment]
I'm happy to announce that it has been fixed in v6.2.0
Again thank you for sharing @AngelFraga
